dependencies.gradle

```diff
@@ -1,11 +1,11 @@
-ext.composeVersion = "1.0.0-alpha07"
+ext.composeVersion = "1.0.0-alpha10"
```

Comment:
Use the most recent version

Reply:
Tried that yesterday, it triggered more updates and in the end functionality disappeared, so I reverted it and called it a day.